.circleci/config.yml

```diff
@@ -124,6 +124,7 @@ job_definitions:
               for i in "${sites[@]}"
               do
                 trigger-build.sh git@github.com:greenpeace/planet4-$i.git << parameters.pipeline >>
+                sleep 5
```

Comment:
I guess 5 won't be enough, but would be nice if it was. It adds up to roughly 5 minutes of delay before the last workflow starts.

Reply:
If it's not much more than 5, we could use this both on production and dev sites. It wouldn't add much time and could also in total speed up total completion.

Reply:
Isn't that 5 seconds?

Reply:
Indeed, seems too low or too high?

Reply:
I think it's too low. Well anything we choose is a bit arbitrary, but let's try with `sleep 30`?

Reply:
Sure, let's start there. I suggest we try a few values to find the sweet spot. If a lower value works equally well we can still shave off almost half an hour of the total deploy times.